backend/python/app/__init__.py

```diff
@@ -1,26 +1,26 @@
 import os
-
 from flask import Flask
-from flask_sqlalchemy import SQLAlchemy
-
+# from flask_sqlalchemy import SQLAlchemy
 from .config import app_config
 
 
 def create_app(config_name):
     app = Flask(__name__)
     app.config.from_object(app_config[config_name])
-    app.config[
-        "SQLALCHEMY_DATABASE_URI"
-    ] = "postgres://{username}:{password}@{host}:5432/{db}".format(
-        username=os.getenv("POSTGRES_USER"),
-        password=os.getenv("POSTGRES_PASSWORD"),
-        host=os.getenv("DB_HOST"),
-        db=os.getenv("POSTGRES_DB"),
-    )
-    app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+    app.config["MONGODB_SETTINGS"] = {"host": os.getenv("MG_DATABASE_URL")}
+    # app.config[
+    #     "SQLALCHEMY_DATABASE_URI"
+    # ] = "postgres://{username}:{password}@{host}:5432/{db}".format(
+    #     username=os.getenv("POSTGRES_USER"),
+    #     password=os.getenv("POSTGRES_PASSWORD"),
+    #     host=os.getenv("DB_HOST"),
+    #     db=os.getenv("POSTGRES_DB"),
+    # )
+    # app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
 
-    from . import models
+    from . import models, routes
```

Comment:
Note that I recently renamed the `routes` module/folder to `rest` (to match the TypeScript backend and for better organization if/when we support Python with GraphQL)